src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -9,13 +9,14 @@
 
 from azure.cli.testsdk.scenario_tests import AllowLargeResponse, live_only
 from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck)
-
+from msrestazure.tools import parse_resource_id
 
 TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
 
 class ContainerappIdentityTests(ScenarioTest):
     @AllowLargeResponse(8192)
     @ResourceGroupPreparer(location="eastus2")
+    @live_only()  # encounters 'CannotOverwriteExistingCassetteException' only when run from recording (passes when run live)
```

Comment:
> 'CannotOverwriteExistingCassetteException'
Will it fail to execute this test locally in record mode? Or it failed only in CI pipeline?
Could you please show us the specific error message?

Reply:
Early this test was passing when run live locally but failing when run from the recording locally. I just tried it again and it seems to work.

I've added the recording back and removed `@live_only()`

Reply:
Actually it looks like these are failing the CI again with `CannotOverwriteExistingCassetteException` (despite passing locally). Can I go ahead and mark these as `@live_only` again @zhoxing-ms?

Reply:
per @panchagnula's comment I'm going to go ahead and mark this test as `@live_only`

Reply:
@StrawnSC yes mark them live for now & lets figure out these flaky tests with upcoming changes.